src/filterConfig.js

```diff
@@ -115,7 +115,7 @@ export const instanceIndexes = [
   // instead. It might make sense to rename the keyword option to something like `keywordAll`
   // but, without tracing the use of the value, I don't know what effects that would have in the code.
   { label: 'ui-inventory.search.all', value: 'all', queryTemplate: 'keyword all "%{query.query}" or isbn="%{query.query}"' },
-  { label: 'ui-inventory.contributor', value: 'contributor', queryTemplate: 'contributors="%{query.query}"' },
+  { label: 'ui-inventory.contributor', value: 'contributor', queryTemplate: 'contributors=="%{query.query}"' },
```

Comment:
Pavlo mentioned that we should use `==/string` operator. It's a little different from just `==`
https://github.com/folio-org/mod-search#cql-query-modifiers

Reply:
Thank you, fixed